apiary-metastore-listener/README.md

```diff
@@ -25,9 +26,13 @@ topic:
 |`eventType`|String|Always|One of: CREATE_TABLE, DROP_TABLE, ALTER_TABLE, ADD_PARTITION, DROP_PARTITION, ALTER_PARTITION| 
 |`dbName`|String|Always|The Hive database name|
 |`tableName`|String|Always|The Hive table name|
+|`tableLocation`|String|In all events except INSERT|The Hive table location|
+|`tableParameters`|String|In all events except INSERT|The Hive table parameters|
```

Comment:
In all events except INSERT if TABLE_PARAM_FILTER configuration value set.

Reply:
No. It will still be there but will be empty even if the TABLE_PARAM_FILTER is not set.

Reply:
Hmmm, given we have potential limits on the SNS message size can't we just not emit at all in this case rather than it be empty?

Reply:
The size of message should not get affected much by an empty map (~20 chars) right ? Also, this extra logic to handle null tableParameters will then have to be added in all places where we will use these tableParameters. That is a `if not null` check wherever we use this map. If you still think that should be removed, I am happy to remove it when TABLE_PARAM_FILTER is not set.

We are in that case having a variable structure of events which can also be confusing to some people.

Reply:
`tableParameters` will now be emitted only when the environment variable is set.